core/src/components/avatar/avatar.ionic.scss

```diff
@@ -22,6 +22,8 @@
 }
 
 :host(:not(.avatar-image)) {
+  @include padding(var(--padding-top), var(--padding-end), var(--padding-bottom), var(--padding-start));
```

Comment:
I realized the padding wasn't actually being applied for this component & we can't apply it when an image is slotted or else the image will be skewed.